### PR TITLE
Adds highlighting to backlogged releases.

### PR DESCRIPTION
**Pull request by rjchaveiro**
Intent
----------------------------
Highlight the release story when it's projected release date have been compromised. 
in accord with #184

### CODE REVIEW

File: config/locales/pt-BR.yml [hunk 1]
```diff
@@ -91,6 +91,8 @@ pt-BR:
     tasks: "Tarefas"
     notes: "Notas"
     attachments: "Anexos"
+    warnings:
+      backlogged_release: "Esse lançamento está com risco de ser completo depois da data de lançamento definida."
```
Comment (rjchaveiro) [inline config/locales/pt-BR.yml]: Special attention to this translation, it might (and probably is) wrong, grammatically.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,9 @@
 ### Changed
 - Update cm42-central-support version to save delivered_at
 
+### Added
+- Highlights to release stories if the release date is compromised.
```
Comment (fladson) [inline CHANGELOG.md]: maybe `Highlights the release stories if the release date is compromised.`

Comment (rjchaveiro) [reply CHANGELOG.md]: That does sounds good.
The idea with the `to` is to reading the `added` first. thus becoming more like:
Added highlights to the release stories if the release dade is compromised.
What do you think?

File: app/assets/javascripts/views/story_view.js [hunk 3]
```diff
@@ -883,6 +884,31 @@ module.exports = FormView.extend({
...
+          }
+        })
+      });
+      return backlogged;
+    }
+  },
+
+  handleBackLoggedRelease: function() {
+    if(this.backLoggedRelease()){
+      this.$el.addClass('backlogged-release');
```
Comment (leonardofreitass) [inline app/assets/javascripts/views/story_view.js]: You could simplify this conditional using `toggleClass`, like:
```js
this.$el.toggleClass('backlogged-release', this.backLoggedRelease());
if (this.backLoggedRelease()) {
  this.$el.attr('title', I18n.t('story.warnings.backlogged_release'));
}
```

Comment (leonardofreitass) [reply app/assets/javascripts/views/story_view.js]: Also, avoid using else, most of the time you can achieve the same logic using just `if` and `return`

File: app/assets/stylesheets/_screen.scss [hunk 4]
```diff
@@ -272,6 +272,12 @@ div.story-title abbr.initials {
   border-bottom: 1px solid $lightyellow-2;
 }
 
+.story.backlogged-release{
+  background-color: rgba($darkred, .5) !important;
+  border-top-color: rgba($darkred, .6)!important;
+  border-bottom-color: rgba($darkred, .1)!important;
```
Comment (leonardofreitass) [inline app/assets/stylesheets/_screen.scss]: Missing space before the `!important`, here and in the line above :nail_care:

File: app/assets/javascripts/views/story_view.js [hunk 5]
```diff
@@ -883,6 +884,29 @@ module.exports = FormView.extend({
     $('.popover').remove();
   },
 
+  backLoggedRelease: function() {
+    let backlogged = false;
+    if(typeof this.model.collection.project.iterations !== 'undefined') {
```
Comment (marclerodrigues) [inline app/assets/javascripts/views/story_view.js]:
```javascript
if(this.model.collation.project.iterations) {
...
}
```

Doesn't work here?

Comment (rjchaveiro) [reply app/assets/javascripts/views/story_view.js]: It works, using typeof is a more reliable way to check only for undefined, although a bit ugly looking if you ask me.

Comment (rjchaveiro) [reply app/assets/javascripts/views/story_view.js]: Haha, yeah! I'm going to change it, because... 💅

**Comment by marclerodrigues** [comment]
Please, squash your commits. :smile:

**Pull request merged**